src/Api/Domain.php

```diff
@@ -84,12 +86,13 @@ public function show(string $domain)
      * @param bool     $wildcard           domain will accept email for subdomains
      * @param bool     $forceDkimAuthority force DKIM authority
      * @param string[] $ips                an array of ips to be assigned to the domain
-     * @param ?string  $pool_id            pool id to assign to the domain
+     * @param ?string  $poolId             pool id to assign to the domain
      * @param string   $webScheme          `http` or `https` - set your open, click and unsubscribe URLs to use http or https. The default is http
+     * @param string   $dkimKeySize        Set length of your domain’s generated DKIM key
      *
      * @return CreateResponse|array|ResponseInterface
      */
-    public function create(string $domain, string $smtpPass = null, string $spamAction = null, bool $wildcard = null, bool $forceDkimAuthority = null, ?array $ips = null, ?string $pool_id = null, string $webScheme = 'http')
+    public function create(string $domain, string $smtpPass = null, string $spamAction = null, bool $wildcard = null, bool $forceDkimAuthority = null, ?array $ips = null, ?string $poolId = null, string $webScheme = 'http', string $dkimKeySize = '1024')
```

Comment:
This would be a breaking change for people using named parameters

Reply:
i see then i guess i will move back underscore

Reply:
I am not sure if named parameters are part of BC promise, maintainers of this project will know. But I wouldn't make any code style changes in a feature PR